### ISSUE TITLE
Constant wind direction does no longer work

### ISSUE THREAD

**Issue by knutfrode** [nansencenter/openwind#16]
Providing constant wind direction does no longer work:
```
sar_wind.py -s RS2/RS2_20150115_072025_0076_SCWA_HHHV_SGF_372273_0708_10847173/ -f test.jpeg -w 90
```
...
```
IOError: Set by user (90): File does not exist
```
This functionality seems to have been broken by https://github.com/nansencenter/openwind/commit/7d294827ec3eae205818bb1deca4c224b72f6953

**Comment by mortenwh** [comment]
which branch is this? did you try it on the release version in master?

**Comment by mortenwh** [comment]
I see I have messed up quite a bit in this commit - must have been in a
hurry. I'll fix it but in the meantime this should work well in the master
branch.

**Comment by mortenwh** [comment]
@knutfrode - can you have a look at the issue16 branch and tell me if it's ok? I have it all in my local develop branch, so will push that if it is ok...

**Comment by mortenwh** [comment]
Fixed and merged into develop at 19e8f1b3acb82942a27f94be933a520bb7115c40
